libs/api/domains/auth/src/lib/resolvers/delegationScope.resolver.ts

```diff
@@ -15,4 +18,18 @@ export class DelegationScopeResolver {
   resolveName(@Parent() delegationScope: DelegationScopeDTO): string {
     return delegationScope.scopeName
   }
+
+  @ResolveField('apiScope', () => ApiScope, { nullable: true })
+  resolveApiScope(
+    @Loader(ApiScopeLoader) apiScopeLoader: ApiScopeDataLoader,
+    @Parent() delegationScope: DelegationScope,
+    @Args('lang', { type: () => String, nullable: true, defaultValue: 'is' })
```

Comment:
I realize it might be a big refactor change throughout the project, but maybe in the future it would be nice to have typed language codes

Reply:
Agree. I think we have it in some places. Just kept it like this for internal consistency.